typescript/infra/scripts/get-typical-remote-gas-amounts.ts

```diff
@@ -0,0 +1,49 @@
+import { ChainMap } from '@hyperlane-xyz/sdk';
+import { stringifyObject } from '@hyperlane-xyz/utils';
+
+import { getOverheadWithOverrides } from '../config/environments/mainnet3/igp.js';
+import { getTypicalRemoteGasAmount } from '../src/config/gas-oracle.js';
+
+import { getArgs } from './agent-utils.js';
+import { getEnvironmentConfig } from './core-utils.js';
+
+// This script exists to print the typical local -> remote gas amounts for a given environment.
+// This is useful for Jake to use in his own models for assessing message costs.
+
+async function main() {
+  const args = await getArgs().argv;
+
+  if (args.environment !== 'mainnet3') {
```

Comment:
nit: do we not have an enum for this?

Reply:
I think this is something you called out a couple months ago as well - we don't and the reason was that it's sorta tough to do without a gnarly cyclic dependency. Happy to have someone add it at some point but I spent > 1hr previously to no avail